drivers/flash/flash_mcux_flexspi_mx25um51345g.c

```diff
@@ -236,7 +236,7 @@ static int flash_flexspi_nor_erase_sector(const struct device *dev,
 		.dataSize = 0,
 	};
 
-	LOG_DBG("Erasing sector at 0x%08x", offset);
+	LOG_DBG("Erasing sector at 0x%08zx", (size_t) offset);
```

Comment:
IMHO it  should be `ssize_t` because `off_t` is signed.
This should never happen, but in case when `off_t` is represented by shorter type then `size_t`, offset that may be negative, due to casting to `ssize_t` will become positive due to lack of sign extension in `off_t`->`size_t` casting: you will print different number than it actually is.

Reply:
@de-nordic thanks - changed to `ssize_t`